sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/PasswordCredentialImpl.java

```diff
@@ -167,4 +175,10 @@ public String id() {
     public String name() {
         return this.name;
     }
+
+    @Override
+    public PasswordCredentialImpl<T> withPasswordConsumer(Consumer<? super PasswordCredential> passwordConsumer) {
```

Comment:
Just `Consumer<PasswordCredential>` would do?

Reply:
`? super` is for inherit, copy from `List.forEach`